measure/endurance-snapshot

```diff
@@ -475,8 +467,14 @@ fi
 echo "- Disk usage"
 LC_ALL=C df -k -P > $path/df
 
-echo "- statefs"
-dump_statefs_value namespaces/Screen/Blanked
+echo "- Blanked state"
+blanked_state=$(qdbus --system com.nokia.mce /com/nokia/mce/request com.nokia.mce.request.get_display_status)
```

Comment:
Should we Add some requirement to .spec file for qdbus?

Reply:
Also if com.nokia.mce does not exist, this will print some error every time endurance-snapshot is executed. Can you add "2>/dev/null" or only call this if com.nokia.mce is actually available?

Reply:
Thanks @mkosola; I've added that in. Thanks also @rantala, that's a good point. Unfortunately it seems qdbus prints its errors to stdout :( so I'll look into the second approach you suggested.

Reply:
Blank/unblank goes through mce - which also gives ui permission to draw -> if mce is not available device will not boot to ui etc -> I would not worry too much about it.

If we want to prepare for such thing anyway, fallback to "blanked=0" might be appropriate as it ought to be less desirable situation than blanked=1.

Reply:
I'm using sp-endurance on x86 server machines, which do not have mce ...

Reply:
I've just pushed a new approach that relies only on mce for the display state. If there's no mce, no graph should be produced, which I hope is okay in your case @rantala if you're not interested in the display state anyway.